### PR TITLE
Panac/fix/sprint 03 07

### LINKED ISSUE #187: Missing license header in a few Typescript files
`cd readium-desktop && node ../r2-utils-js/tools/license_header.js ./src/` (assumes `r2-utils-js` is a local sibling folder to the `readium-desktop` repository clone)

In branch `feature/new-db`:

```
src/common/models/dialog.ts
src/common/models/font.ts
src/common/models/locator.ts
src/common/models/timestampable.ts
src/common/redux/actions/api.ts
src/common/redux/actions/dialog.ts
src/common/redux/reducers/dialog.ts
src/common/redux/states/dialog.ts
src/common/views/catalog.ts
src/common/views/publication.ts
src/main/api/catalog.ts
src/main/api/publication.ts
src/main/converter/navigator.ts
src/main/converter/publication.ts
src/main/db/document/config.ts
src/main/db/document/locator.ts
src/main/db/document/opds.ts
src/main/db/document/publication.ts
src/main/db/exceptions.ts
src/main/redux/sagas/api.ts
src/renderer/components/PageManager.tsx
src/renderer/components/SecondaryHeader.tsx
src/renderer/components/catalog/AddEntryForm.tsx
src/renderer/components/catalog/Catalog.tsx
src/renderer/components/catalog/GridView.tsx
src/renderer/components/catalog/Header.tsx
src/renderer/components/catalog/ListView.tsx
src/renderer/components/catalog/SearchForm.tsx
src/renderer/components/dialog/Dialog.tsx
src/renderer/components/dialog/DialogManager.tsx
src/renderer/components/dialog/FileImport.tsx
src/renderer/components/layout/BreadCrumb.tsx
src/renderer/components/layout/LibraryHeader.tsx
src/renderer/components/layout/LibraryLayout.tsx
src/renderer/components/opds/Header.tsx
src/renderer/components/opds/SearchForm.tsx
src/renderer/components/publication/PublicationInfo.tsx
src/renderer/components/publication/TagManager.tsx
src/renderer/components/reader/ReaderFooter.tsx
src/renderer/components/reader/ReaderHeader.tsx
src/renderer/components/reader/ReaderMenu.tsx
src/renderer/components/reader/ReaderOptions.tsx
src/renderer/components/searchResult/AllPublicationPage.tsx
src/renderer/components/searchResult/GridView.tsx
src/renderer/components/searchResult/ListView.tsx
src/renderer/components/searchResult/TagSearchResult.tsx
src/renderer/components/searchResult/TextSearchResult.tsx
src/renderer/components/settings/CatalogEntrySettings.tsx
src/renderer/components/settings/Header.tsx
src/renderer/components/settings/LanguageSettings.tsx
src/renderer/components/utils/Dialog.tsx
src/renderer/components/utils/DragAndDropList.tsx
src/renderer/components/utils/SVG.tsx
src/renderer/components/utils/Slider.tsx
src/renderer/components/utils/api.tsx
src/renderer/components/utils/menu/Menu.tsx
src/renderer/components/utils/menu/MenuButton.tsx
src/renderer/components/utils/menu/MenuContent.tsx
src/renderer/components/utils/translator.tsx
src/renderer/redux/reducers/api.ts
src/renderer/redux/states/api.ts
src/renderer/routing.ts
src/utils/debug.ts
src/utils/fontList.ts
```

### PR DESCRIPTION
**Pull request by panaC**
See commit name

fix #187 also

### CODE REVIEW

File: src/resources/locales/en.json [hunk 1]
```diff
@@ -211,7 +211,7 @@
         "annotations": "Annotations"
       },
       "svg": {
-        "contentTable": "Content Table",
+        "contentTable": "Navigation Table",
```
Comment (danielweck) [inline src/resources/locales/en.json]: A couple of questions:
* Where is this label used?
* Should this be "table of contents"? (I'm not sure, depends where the label is used)

File: src/renderer/components/reader/ReaderOptions.tsx [hunk 2]
```diff
@@ -139,7 +139,7 @@ export class ReaderOptions extends React.Component<Props> {
                     <span className={styles.slider_marker} >a</span>
                     <input type="range"
                         onChange={(e) => this.props.handleIndexChange(e, "fontSize")}
-                        id="taille_texte"
+                        id="text_length"
```
Comment (danielweck) [inline src/renderer/components/reader/ReaderOptions.tsx]: The point of issue https://github.com/readium/readium-desktop/issues/218 was that the ID is duplicated in the markup.

File: src/resources/locales/en.json (hunk 1 above)
Comment (panaC) [reply src/resources/locales/en.json]: cf Fernando meeting,
@llemeurfr ?

File: src/renderer/components/reader/ReaderOptions.tsx (hunk 2 above)
Comment (danielweck) [reply src/renderer/components/reader/ReaderOptions.tsx]: New issue, to be addressed outside of this PR: https://github.com/readium/readium-desktop/issues/398

File: src/resources/locales/en.json (hunk 1 above)
Comment (danielweck) [reply src/resources/locales/en.json]: New issue to be addressed separately https://github.com/readium/readium-desktop/issues/396

**Pull request merged**